mlflow/projects/__init__.py

```diff
@@ -364,7 +364,7 @@ def _create_run(uri, experiment_id, work_dir, entry_point, parameters):
     used to report additional data about the run (metrics/params) to the tracking server.
     """
     active_run = tracking._create_run(
-        experiment_id=experiment_id, source_name=_expand_uri(uri),
+        experiment_id=experiment_id, source_name=tracking._get_git_url(_expand_uri(uri)),
```

Comment:
We should probably only call this if `_is_local_uri(uri)`, right? Or you can add that check in `_get_git_uri` itself.

Reply:
Done